src/ui/conatiner/Container.tsx

```diff
@@ -0,0 +1,12 @@
+import { CardContent, Paper } from '@mui/material';
+
+import { styles } from './Container.styles';
+import { ContainerProps } from './Container.types';
+
+export const Container = ({ children }: ContainerProps) => {
```

Comment:
Wiesz co, Container to bardziej taki niewidzialny element, ewentualnie container jako komponent z komunikacją z API. Co powiesz na `Card`?